packages/cli/test/with-context.js

```diff
@@ -0,0 +1,50 @@
+/** @import {Context, TestRoutine} from '../types' */
+/**
+ * Creates a wrapper which wraps a test routine with an execa-curried setup and teardown routine.
+ *
+ * @param {Context} context
+ * @returns {(testRoutine: TestRoutine) => TestRoutine}
+ */
+const makeContextWrapper =
+  context => testRoutine => async (execa, testCommand) => {
+    await null;
+    try {
+      await context.setup(execa);
+      await testRoutine(execa, testCommand);
+    } finally {
+      (await context.teardown) ?? execa;
```

Comment:
What's the purpose of `?? execa` here? Unless I'm misreading this, it does not appear to do anything.

Reply:
silly me, I used the `??` rune where `?.` was required